test/src/unit/repo_test_utils.h

```diff
@@ -237,13 +238,11 @@ static bool projectHasGeometryWithMetadata(std::string dbName, std::string proje
 					// toString will stringify the underlying type of the value, in the same
 					// way it is stringified for the frontend.
 
-					auto aValue = entry.toMongoElement().Obj().getField(REPO_NODE_LABEL_META_VALUE).toString(false);
-
+					auto aValue = entry.toMongoElement().Obj().getField(REPO_NODE_LABEL_META_VALUE).String();
```

Comment:
not sure why this is changed

We haven't really changed the way we store the field in mongo?

Reply:
This change was not necessary this was done for checking purpose hence reverting this changes now.